Gems/LyShine/Code/Source/UiCanvasComponent.cpp

```diff
@@ -3609,83 +3627,25 @@ void UiCanvasComponent::CreateRenderTarget()
         return;
     }
 
-#ifdef LYSHINE_ATOM_TODO // [GHI #6269] Support RTT using Atom
-    // Create a render target that this canvas will be rendered to.
-    // The render target size is the canvas size.
-    m_renderTargetHandle = gEnv->pRenderer->CreateRenderTarget(m_renderTargetName.c_str(),
-            static_cast<int>(m_canvasSize.GetX()), static_cast<int>(m_canvasSize.GetY()), Clr_Empty, eTF_R8G8B8A8);
-
-    if (m_renderTargetHandle <= 0)
+    // Create a render target for the canvas
+    AZ::RHI::Size imageSize(static_cast<uint32_t>(m_canvasSize.GetX()), static_cast<uint32_t>(m_canvasSize.GetY()), 1);
+    EBUS_EVENT_ID_RESULT(m_attachmentImageId, GetEntityId(), LyShine::RenderToTextureRequestBus, UseRenderTarget, AZ::Name(m_renderTargetName.c_str()), imageSize);
+    if (m_attachmentImageId.IsEmpty())
     {
-        gEnv->pSystem->Warning(VALIDATOR_MODULE_SHINE, VALIDATOR_WARNING, VALIDATOR_FLAG_FILE,
-            m_pathname.c_str(),
-            "Failed to create render target for UI canvas: %s",
-            m_pathname.c_str());
-    }
-    else
-    {
-        // Also create a depth surface to render the canvas to, we need depth for masking
-        // since that uses the stencil buffer
-        m_renderTargetDepthSurface = gEnv->pRenderer->CreateDepthSurface(
-                static_cast<int>(m_canvasSize.GetX()), static_cast<int>(m_canvasSize.GetY()));
-
-        ISystem::CrySystemNotificationBus::Handler::BusConnect();
+        AZ_Warning("UI", false, "Failed to create render target for UI canvas: %s", m_pathname.c_str());
```

Comment:
nit: Since the second parameter was meant to gate warnings anyways, maybe you don't need the surrounding `if` if there is no other action if `m_attachmentImageId.IsEmpty()` is true

```c++
AZ_Warning("UI", !m_attachmentImageId.IsEmpty(), "Failed to create render target for UI canvas: %s", m_pathname.c_str());
```